packages/components/src/Header/Header.spec.js

```diff
@@ -0,0 +1,85 @@
+import { render } from '@testing-library/react';
+import { ThemeProvider, createTheme } from '@platzily-ui/styling';
+import Header from './Header';
+
+describe('@Components/Header', () => {
+  const spy = {};
+
+  beforeEach(() => {
+    spy.console = jest.spyOn(console, 'error').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    spy.console.mockRestore();
+  });
+
+  it('Given a styling by style attribute using theme with createTheme() ', () => {
+    // arrange
+    const theme = createTheme();
+    const { getByRole } = render(
+      <ThemeProvider theme={theme}>
+        <Header role="banner" style={{ color: theme.palette.secondary.main }} />
+      </ThemeProvider>,
+    );
+
+    // act
+    const HeaderTestedText = getByRole('banner');
+    // assert
+    expect(HeaderTestedText).toBeDefined();
+    expect(HeaderTestedText).toHaveStyle(`color:#97c343`);
+    expect(spy.console).not.toBeCalled();
+  });
+
+  it('Given a styling by style attribute using one color directly ', () => {
```

Comment:
Same as above with the utterance. What are we testing? This test looks like we are testing the same thing as before :thinking: